mlir/utils/tuna/tuna-script.sh

```diff
@@ -0,0 +1,155 @@
+#!/bin/bash
+
+# For installing mysql 8.0 for testing, or for running with an isolated database.
+function mysql_setup_generic
+{
+    # Latest "generic" version.  (Must have libaio1 installed already.)
+    # Note that all this happens without privileges.
+    pushd /tmp
+    wget https://dev.mysql.com/get/Downloads/MySQL-8.0/mysql-8.0.34-linux-glibc2.28-x86_64.tar.gz
+    tar xf mysql-8.0.34-linux-glibc2.28-x86_64.tar.gz
+    mv mysql-8.0.34-linux-glibc2.28-x86_64 mysql
+    export PATH=$PATH:/tmp/mysql/bin
+    mysqld --initialize-insecure --datadir=/tmp/mysql/data
```

Comment:
More accurately, I'd say the wget should be in the Dockerfile, and then we should create the database, either under /tmp/ or, more interestingly, under $WORKSPACE.